src/main/Validator.js

```diff
@@ -23,8 +25,8 @@ const argsSchema = Joi.object({
 });
 
 const boundsInfoSchema = Joi.object({
-  x: Joi.number().integer().min(0),
-  y: Joi.number().integer().min(0),
+  x: Joi.number().integer().min(defaultMinWindowXPos).default(0),
+  y: Joi.number().integer().min(defaultMinWindowYPos).default(0),
```

Comment:
This was changed to prevent the window size from resetting when maximized in Windows as the reported x/y coordinate when maximized in Windows is -8/-8.